[2025-04-18T18:31:16.941+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T18:31:16.973+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_run manual__2025-04-18T18:27:08.156311+00:00 [queued]>
[2025-04-18T18:31:16.992+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_run manual__2025-04-18T18:27:08.156311+00:00 [queued]>
[2025-04-18T18:31:16.995+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-04-18T18:31:17.025+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): dbt_tasks.dbt_run> on 2025-04-18 18:27:08.156311+00:00
[2025-04-18T18:31:17.048+0000] {standard_task_runner.py:72} INFO - Started process 2469 to run task
[2025-04-18T18:31:17.054+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_api_to_snowflake', 'dbt_tasks.dbt_run', 'manual__2025-04-18T18:27:08.156311+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/dbt_snowflake_pipeline.py', '--cfg-path', '/tmp/tmp9vicr4g3']
[2025-04-18T18:31:17.068+0000] {standard_task_runner.py:105} INFO - Job 276: Subtask dbt_tasks.dbt_run
[2025-04-18T18:31:17.197+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_run manual__2025-04-18T18:27:08.156311+00:00 [running]> on host b322df8cb9fb
[2025-04-18T18:31:17.356+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_to_snowflake' AIRFLOW_CTX_TASK_ID='dbt_tasks.dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T18:27:08.156311+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T18:27:08.156311+00:00'
[2025-04-18T18:31:17.360+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-18T18:31:17.405+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-18T18:31:17.408+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt run --project-dir /opt/dbt --profiles-dir /opt/dbt']
[2025-04-18T18:31:17.434+0000] {subprocess.py:99} INFO - Output:
[2025-04-18T18:31:23.271+0000] {subprocess.py:106} INFO - [0m18:31:23  Running with dbt=1.9.4
[2025-04-18T18:31:26.668+0000] {subprocess.py:106} INFO - [0m18:31:26  Registered adapter: snowflake=1.9.2
[2025-04-18T18:31:29.346+0000] {subprocess.py:106} INFO - [0m18:31:29  Found 2 models, 4 data tests, 1 source, 474 macros
[2025-04-18T18:31:29.357+0000] {subprocess.py:106} INFO - [0m18:31:29
[2025-04-18T18:31:29.361+0000] {subprocess.py:106} INFO - [0m18:31:29  Concurrency: 1 threads (target='dev')
[2025-04-18T18:31:29.365+0000] {subprocess.py:106} INFO - [0m18:31:29
[2025-04-18T18:31:30.817+0000] {subprocess.py:106} INFO - [0m18:31:30  1 of 2 START sql view model BASE.stg_weather_raw ............................... [RUN]
[2025-04-18T18:31:31.170+0000] {subprocess.py:106} INFO - [0m18:31:31  1 of 2 OK created sql view model BASE.stg_weather_raw .......................... [[32mSUCCESS 1[0m in 0.34s]
[2025-04-18T18:31:31.180+0000] {subprocess.py:106} INFO - [0m18:31:31  2 of 2 START sql table model BASE.weather_summary .............................. [RUN]
[2025-04-18T18:31:32.447+0000] {subprocess.py:106} INFO - [0m18:31:32  2 of 2 OK created sql table model BASE.weather_summary ......................... [[32mSUCCESS 1[0m in 1.26s]
[2025-04-18T18:31:32.574+0000] {subprocess.py:106} INFO - [0m18:31:32
[2025-04-18T18:31:32.577+0000] {subprocess.py:106} INFO - [0m18:31:32  Finished running 1 table model, 1 view model in 0 hours 0 minutes and 3.21 seconds (3.21s).
[2025-04-18T18:31:32.754+0000] {subprocess.py:106} INFO - [0m18:31:32
[2025-04-18T18:31:32.756+0000] {subprocess.py:106} INFO - [0m18:31:32  [32mCompleted successfully[0m
[2025-04-18T18:31:32.759+0000] {subprocess.py:106} INFO - [0m18:31:32
[2025-04-18T18:31:32.762+0000] {subprocess.py:106} INFO - [0m18:31:32  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2025-04-18T18:31:33.843+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-18T18:31:33.900+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-18T18:31:33.903+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_api_to_snowflake, task_id=dbt_tasks.dbt_run, run_id=manual__2025-04-18T18:27:08.156311+00:00, execution_date=20250418T182708, start_date=20250418T183116, end_date=20250418T183133
[2025-04-18T18:31:34.000+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T18:31:34.051+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-18T18:31:34.056+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
